data-es7.js

```diff
@@ -479,56 +438,596 @@ exports.tests = [
+      */},
+      res: {
+        firefox39: true,
+        chrome41:  false
+      }
+    },
+    'SIMD.%type%.lessThan' : {
+      exec: function(){/*
+        return typeof SIMD.float32x4.lessThan === 'function';
+      */},
+      res: {
+        firefox39: true,
+        chrome41:  false
+      }
+    },
+    'SIMD.%type%.lessThanOrEqual' : {
+      exec: function(){/*
+        return typeof SIMD.float32x4.lessThanOrEqual === 'function';
+      */},
+      res: {
+        firefox39: true,
+        chrome41:  false
+      }
+    },
+    'SIMD.%type%.mul' : {
+      exec: function(){/*
+        return typeof SIMD.float32x4.mul === 'function';
+      */},
+      res: {
+        firefox39: true,
+        chrome41:  false
+      }
+    },
+    'SIMD.%type%.neg' : {
+      exec: function(){/*
+        return typeof SIMD.float32x4.neg === 'function';
+      */},
+      res: {
+        firefox39: true,
+        chrome41:  false
+      }
+    },
+    'SIMD.%type%.not' : {
+      exec: function(){/*
+        return typeof SIMD.float32x4.not === 'function';
+      */},
+      res: {
+        firefox39: true,
+        chrome41:  false
+      }
+    },
+    'SIMD.%type%.notEqual' : {
+      exec: function(){/*
+        return typeof SIMD.float32x4.notEqual === 'function';
+      */},
+      res: {
+        firefox39: true,
+        chrome41:  false
+      }
+    },
+    'SIMD.%type%.or' : {
+      exec: function(){/*
+        return typeof SIMD.float32x4.or === 'function';
+      */},
+      res: {
+        firefox39: true,
+        chrome41:  false
+      }
+    },
+    'SIMD.%type%.select' : {
+      exec: function(){/*
+        return typeof SIMD.float32x4.select === 'function';
+      */},
+      res: {
+        firefox39: true,
+        chrome41:  false
+      }
+    },
+    'SIMD.%type%.shuffle' : {
+      exec: function(){/*
+        return typeof SIMD.float32x4.shuffle === 'function';
+      */},
+      res: {
+        firefox39: true,
+        chrome41:  false
+      }
+    },
+    'SIMD.%type%.sub' : {
+      exec: function(){/*
+        return typeof SIMD.float32x4.sub === 'function';
+      */},
+      res: {
+        firefox39: true,
+        chrome41:  false
+      }
+    },
+    'SIMD.%type%.swizzle' : {
+      exec: function(){/*
+        return typeof SIMD.float32x4.swizzle === 'function';
+      */},
+      res: {
+        firefox39: true,
+        chrome41:  false
+      }
+    },
+    'SIMD.%type%.xor' : {
+      exec: function(){/*
+        return typeof SIMD.float32x4.xor === 'function';
+      */},
+      res: {
+        firefox39: true,
+        chrome41:  false
+      }
+    }
+  }
+},
+{
+  name: 'Class and Property Decorators',
+  category: 'proposal',
+  link: 'https://github.com/wycats/javascript-decorators',
+  exec: function(){/*
+    class Person {
+      constructor(){
+        this.children = ['Tom', 'Dick', 'Harry'];
+      }
+      @nonenumerable
+      get kidCount() { return this.children.length; }
```

Comment:
@zloirock I'll make a better test - somehow I got this test to work as expected in babel.js last night but obviously the test is broken having looked over the proposal again.